### ISSUE TITLE
Awkward jumps in links

### ISSUE THREAD

**Issue by tzimmermsr** [martinvelez/fse2016#36]
In the middle box (the box that shows "FSE Week Overview"), move the mouse over a link (for example "Conference Preview". When you do this the link jump away from the mouse. 

Can we disable this behavior? It's very strange. It's probably some weird CSS.

**Comment by MonaRahimi** [comment]
Hi Tom, 
I have checked the CSS for sidebar1 and also the custom CSS code. I did not find anything suspicious! but this is a strange behaviour! The only link, in sidebar1, which does not jump is the last one "Hotel reservations now open". But I could not find anything different in the other links. I will look into it more when I get a chance.

**Comment by tzimmermsr** [comment]
Thanks Mona. I appreciate it.

I noticed that the effect happens when there is no paragraph before a heading, so as a workaround I added text before each heading. No need to further investigate. CSS can get very ugly.

If you have a moment, there is this weird animation (text moves to the right) when hovering over the “Room Sharing” in the last paragraphs. This is low priority, don’t spend too much time on it (max 5-10 minutes).

--Tom
